src/components/ReplyComment/index.js

```diff
@@ -66,6 +67,8 @@ const ReplyCommentId = ({
     }
   }, [temporaryText]);
 
+  const updateComment = () => getThisComment(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateComment` function is introduced which calls `getThisComment(true)`. Make sure that `getThisComment` can handle the boolean argument correctly and that it doesn't introduce any side effects when called with `true`.
```diff
- 
+   const updateComment = () => getThisComment(true);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->